src/puppetlabs/puppetdb/query/paging.clj

```diff
@@ -216,17 +216,8 @@
     ((some-fn nil? (every-pred coll? empty?)) order_by)
     (not count?))))
 
-(defn rename-first
-  "rename the first element of a vector according to kmap"
-  [kmap pair]
-  (update-in pair [0] #(% kmap)))
-
 (defn dealias-order-by
   [{:keys [projections] :as query-rec} paging-options]
-  (let [alias-map (reduce-kv (fn [acc k v]
-                               (assoc acc
-                                 (keyword k)
-                                 (-> v :field h/extract-sql)))
-                             {} projections)]
-    (update-in paging-options [:order_by]
-               #(map (partial rename-first alias-map) %))))
+  (let [alias-map (->> (kitchensink/mapvals (comp h/extract-sql :field) projections)
```

Comment:
probably shouldn't have picked this time and place to start this discussion. But: especially for someone with imperfect knowledge of the codebase, isolating changes as much as possible to small, interdependent sets that leave the system in a working state is very valuable. Each of them can have its own commit message, in this case 'simplify dealias-order-by' would have been sufficient. That would let any reviewer plus a future history delver know the intent behind the change, which is like 75% of why we do source control in the first place. 

I know this isn't generally what we do, and that I'm asking for a change. I also know that github doesn't support this well. But I feel like we need to figure something out here.